bouncer.go

```diff
@@ -366,56 +393,198 @@ func crowdsecQuery(bouncer *Bouncer, stringURL string) ([]byte, error) {
 }
 
 func validateParams(config *Config) error {
```

Comment:
try to create validate pkg, impossible to keep 500 line in this file

Reply:
and maybe another one for TLS

Reply:
I propose to do a rewrite as a separate issue/mr